### DISCUSSION TITLE
Something like latex' substack

### DISCUSSION THREAD

**Discussion by lacarthur** [typst/typst#4533] [Q&A]
Hi, I'm trying to do the following: 
```
    {      multi        }
A = |      line         |
    {     definition     }
```
in a math block. In Latex I would use something like \substack, but how do I do this in Typst?
Thanks,

**Comment by ludwig-austermann** [comment]
Something like `$sum_(0<=i,j<=n \ i != j)$` probably does what you are looking for. Hence, inside brackets, you can use the `\` to make a “local” line break.

However, the alignment and line spacing are different. Currently, to obtain identical display behaviour as of substack, you would probably need to define your own command via a grid.

**Reply by wrzian** [reply]
You might also try the `cases` function: https://typst.app/docs/reference/math/cases/
